integrationTests/chainSimulator/testing.go

```diff
@@ -18,6 +18,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TriggerChangeOfEpoch -
+func TriggerChangeOfEpoch(t *testing.T, chainSimulator ChainSimulator, nodeHandler chainSimulatorProcess.NodeHandler) {
```

Comment:
This func can definelty fail if called from any other epoch than 0.

We can do it better and iterate from i:= currentEpoch; i< currentEpoch+4